src/base/bittorrent/bencoderesumedatastorage.cpp

```diff
@@ -242,11 +270,40 @@ std::optional<BitTorrent::LoadTorrentParams> BitTorrent::BencodeResumeDataStorag
 }
 
 void BitTorrent::BencodeResumeDataStorage::store(const TorrentID &id, const LoadTorrentParams &resumeData) const
+{
+    QMetaObject::invokeMethod(m_asyncWorker, [this, id, resumeData]()
+    {
+        static_cast<AsyncWorker *>(m_asyncWorker)->store(id, resumeData);
```

Comment:
is `static_cast` still required?

Reply:
No, of course. Just forgot to remove it.
Fixed.